tests/modules/images/domiant-color/get-dominant-color-GD-test.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Tests for webp-uploads module.
+ *
+ * @package performance-lab
+ * @group   webp-uploads
+ */
+
+class get_dominant_color_GD_Tests extends WP_UnitTestCase {
```

Comment:
Once the GD / imagek classes exist, these classes should be renamed and implement covers notes.

Reply:
Let's name these test classes, similar to the class.